Mathlib/Algebra/Associated.lean

```diff
@@ -795,7 +807,7 @@ namespace Associates
 open Associated
 
 /-- The canonical quotient map from a monoid `α` into the `Associates` of `α` -/
-protected abbrev mk {α : Type*} [Monoid α] (a : α) : Associates α :=
```

Comment:
I think this should be an `abbrev`, since `Associates` is also an abbrev.

Reply:
What do you think about making `Associates` a `def` then?

Reply:
Could we leave that (and this) to another PR? It's not clear to me what advantage there is to hiding the fact that `Associates` is defined via a quotient.

Reply:
OK

Reply:
I changed `mk` to `def` and now I can see my reason: `Quotient.eq` simplifies `mk a = mk b` to `a ≈ b` instead of `Associated a b`. Also, we need to make `Associated.setoid` an `instance` here and there.

Reply:
I think this is probably indicative that the simp-normal form around `Quotient` isn't right.

Reply:
What do you suggest?

Reply:
I added `-Quotient.eq` to 2 `simp`s for now.